src/Output/OutputInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Tomaj\NetteApi\Output;
+
+use Tomaj\NetteApi\Response\ResponseInterface;
+
+interface OutputInterface
+{
+    public function validate(ResponseInterface $response): bool;
```

Comment:
tento interface mi pride taky divny

nestalo keby tu bola len metoda validate() ? a ta by vratila mozno nejaky OutputResult alebo nieco take z ktoreho si mozes vybrat errory a spytat sa ho ci je v poriadku? lebo tento interface je taky divny a vyzaduje od teba a by si si nejak menezoval stav (ukladal errory v instancii), co potom emulujeme cez tu abstraktnut triedu zbytocne ktoru by sme mohli zrusit

Reply:
Moze to byt aj tak, chcel som aby validate vracal boolean a neriesil by si nejaky dalsi objekt. Rovnako to robi napr. aj ten schema validator ktory tu pouzivam. Ale ano musis vramci validate pridavat errory do nejakeho error pola (ak chces aby sa errory vypisali)

Reply:
prvy nastrel bol ze validate len zvaliduje output a vrati boolean, lenze ak nastane chyba niekde v jsone tak si mozes aj vlasy vytrhat a neprides na to - preto som pridal tie errory... druha vec je, ze ak mas napr. 10 moznych outputov, tak kazdy z nich zapise error, preco sa nematchol s realnym responsom a potom tam uz tiez dost tazko dohladas realnu chybu, ktora nastala pri tvojom response a ocakavanom outpute... cize toto by som potreboval nejako domysliet este

Reply:
no je to take pomixovane a ten interface ta potom nuti nieco implmenetovat konkretnym sposobom. ja by som zaviedol ten novy objekt kde budu tie errory aj to bude vraciat status

Reply:
jasne :) len som to tak spravil aj v input paramsoch :) to sa asi stratilo v tych milion commitoch co som daval :)

Reply:
prerobene

Reply:
@tomaj stale nie dobre? :)

Reply:
dobre ;-) dikes